dev/prod_builders.json

```diff
@@ -121,10 +121,16 @@
       "flaky": false
     },
     {
-      "name": "Linux docs",
+      "name": "Linux docs_test",
       "repo": "flutter",
-      "task_name": "linux_docs",
-      "flaky": false
+      "task_name": "linux_docs_test",
+      "flaky": true
+    },
+    {
```

Comment:
Should Linux docs_publish run only in release branches? adding it here will run it for master.

Reply:
I think it's supposed to run on master, just not the release candidate branches:
https://github.com/flutter/infra/blob/e83d25832f6b56a7651365076eb842f57328343a/config/packaging_config.star#L149-L152
```
    # Docs builds are only done on master and stable channels
    if branch in ("master", "stable"):
        common.linux_prod_builder(
            name = "Linux%s docs_publish|docs" % ("" if branch == "master" else " " + branch),
```
I'll wait until @christopherfujino comes back from vacation, he can validate that's what he intended.

Reply:
this change is correct, provided we follow-up to mark un-flaky.

confusingly, docs are published only on master (https://master-api.flutter.dev) and stable channels (https://api.flutter.dev).